### PR TITLE
Fix the number of rows of column not correct

### PR DESCRIPTION
**Pull request by yah01**
/kind bug
related #26249

### CODE REVIEW

**Review by jiaoew1991 — APPROVED** [review]
/lgtm

File: internal/core/src/mmap/Column.h [hunk 1]
```diff
@@ -136,7 +167,7 @@ class ColumnBase {
 
         if (data_ != nullptr) {
             std::memcpy(data, data_, len_);
-            if (munmap(data_, cap_)) {
+            if (munmap(data_, cap_ + padding_)) {
                 AssertInfo(
```
Comment (zhagnlu) [inline internal/core/src/mmap/Column.h:171]: Is there some situation that unmap it, but others is still using it ?  Or multi-thread problem

Comment (yah01) [reply internal/core/src/mmap/Column.h:171]: `Append()` is for load only, and the column is immutable after load

File: internal/core/src/mmap/Column.h [hunk 2]
```diff
@@ -145,49 +177,41 @@ class ColumnBase {
         }
 
         data_ = data;
-        cap_ = size;
+        cap_ = new_size;
     }
 
     char* data_{nullptr};
+    // capacity in bytes
     size_t cap_{0};
```
Comment (zhagnlu) [inline internal/core/src/mmap/Column.h:185]: this cap_ actually is not cap but mmap memory, so using max_data_  is better

**Pull request merged**